### ISSUE TITLE
Fish icon not displaying correctly in iTerm2

### ISSUE THREAD

**Issue by pmdaly** [oh-my-fish/oh-my-fish#831]
![Screen Shot 2021-04-23 at 10 11 35 AM](https://user-images.githubusercontent.com/8229664/115908180-accca900-a41e-11eb-856e-59350282b46a.png)

I followed https://github.com/oh-my-fish/oh-my-fish/issues/764 the suggestions here by selecting `Use built-in powerline glyphs` in iterm2 settings and I also installed Nerd Fonts.  

Fresh Fish and OMF installs.  I'm using Solarized Dark iTerm2 theme. 
```
Oh My Fish version:   7
OS type:              Darwin
Fish version:         fish, version 3.2.2
Git version:          git version 2.30.1
Git core.autocrlf:    no
Checking for a sane environment...
Your shell is ready to swim.
```

**Comment by lnx-soul** [comment]
I have experience the same thing in iTerm2 and the default Terminal app in Ubuntu, plus others.

It looks like some of the Powerline fonts aren't rendering correctly because this exists in a slightly different variation in themes like `bobthefish`.  The only way around this for me is to use a theme without any sharp edges, like `spacefish`.

**Comment by pmdaly** [comment]
> I have experience the same thing in iTerm2 and the default Terminal app in Ubuntu, plus others.
> 
> It looks like some of the Powerline fonts aren't rendering correctly because this exists in a slightly different variation in themes like `bobthefish`. The only way around this for me is to use a theme without any sharp edges, like `spacefish`.

The

**Comment by pmdaly** [comment]
> I have experience the same thing in iTerm2 and the default Terminal app in Ubuntu, plus others.
> 
> It looks like some of the Powerline fonts aren't rendering correctly because this exists in a slightly different variation in themes like `bobthefish`. The only way around this for me is to use a theme without any sharp edges, like `spacefish`.

It could just be the way the icon is defined.  If you look at the vi mode icon, it looks very similar.  To resolve the vi mode icon color you just need to change the colors in the prompt or wait for the next patch.  https://github.com/fish-shell/fish-shell/pull/7880

**Comment by lnx-soul** [comment]
Yeah, it's hard to say.  Eventually I found a $SHELL theme that worked fine on Ubuntu.  A lot of the Powerline theme variations just don't line up that well.  I switched to Alacritty and it works a little bit better.  It may be because `$TERM` should be set to `xterm-256color`.
